ACE/ace/DLL_Manager.cpp

```diff
@@ -11,7 +11,7 @@
 #include "ace/Recursive_Thread_Mutex.h"
 #include "ace/Guard_T.h"
 #include "ace/OS_NS_dlfcn.h"
-#include "ace/OS_NS_string.h"
+#include "ace/OS_NS_stdio.h"
```

Comment:
When this include is only for AIX, maybe guard it also with a ifdef check?

Reply:
Good idea. I did not check the rest of the file for a need for OS_NS_stdio.h.